Civi/CCase/Utils.php

```diff
@@ -5,13 +5,36 @@
 
 class Utils {
 
+  /**
+   * Add a case_manager join
+   *
+   * @param \CRM_Utils_SQL_Select $sql
+   * @param String $relationship
+   */
+  public static function joinOnRelationship($sql, $relationship) {
```

Comment:
i don't understand why you moved this `joinXYZ` method above the (now called) `getRelationshipTypes` method, instead of simply amending the old `joinOnManager` method. It creates unnecessary diffs that are harder to review